systemtest/src/test/java/io/strimzi/systemtest/migration/MigrationST.java

```diff
@@ -239,17 +283,64 @@ void testMigrationFromZkToKRaft(ExtensionContext extensionContext) {
         RollingUpdateUtils.waitTillComponentHasRolledAndPodsReady(testStorage.getNamespaceName(), brokerSelector, 3, brokerPodsSnapshot);
         RollingUpdateUtils.waitTillComponentHasRolledAndPodsReady(testStorage.getNamespaceName(), controllerSelector, 3, controllerPodsSnapshot);
 
-        LOGGER.info("13. creating KafkaTopic: {} and checking if the metadata are only in KRaft", kraftTopicName);
-        resourceManager.createResourceWithWait(extensionContext, KafkaTopicTemplates.topic(testStorage.getClusterName(), kraftTopicName, testStorage.getNamespaceName()).build());
+        createKafkaTopicAndCheckMetadataWithMessageTransmission(extensionContext, testStorage, kraftTopicName, false);
+
+        LOGGER.info("Migration is completed, waiting for continuous clients to finish");
+        ClientUtils.waitForClientsSuccess(continuousClients.getProducerName(), continuousClients.getConsumerName(), testStorage.getNamespaceName(), continuousClients.getMessageCount());
+    }
+
+    private void doRollback(TestStorage testStorage) {
+        LOGGER.info("From {} state we are going to roll back to ZK", KafkaMetadataState.KRaftPostMigration.name());
+
+        LOGGER.info("Rolling migration back - applying {}: rollback annotation", Annotations.ANNO_STRIMZI_IO_KRAFT);
```

Comment:
aren't we using something like `applying the {} annotation with value: {}` in other places?